pgx/src/spi.rs

```diff
@@ -171,6 +171,12 @@ impl Spi {
             client.update(query, None, None);
         })
     }
+    
+    fn run_with_args(query: &str, args: Vec<(PgOid, Option<pg_sys::Datum>)>) {
```

Comment:
You will want to make this function `pub` and also add a comment.
```suggestion
    pub fn run_with_args(query: &str, args: Vec<(PgOid, Option<pg_sys::Datum>)>) {
```